server/database/setup.sql

```diff
@@ -0,0 +1,40 @@
+DROP DATABASE IF EXISTS garager;
+CREATE DATABASE garager;
+
+\c garager;
+
+DROP TABLE IF EXISTS users;
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY
+);
+
+
+DROP TABLE IF EXISTS items;
+CREATE TABLE items (
+  item_id SERIAL PRIMARY KEY,
+  user_id INTEGER NOT NULL DEFAULT NULL,
+  title TEXT NOT NULL DEFAULT NULL,
+  category TEXT NOT NULL DEFAULT NULL,
+  brand TEXT NOT NULL DEFAULT NULL,
+  model TEXT NOT NULL DEFAULT NULL,
+  itemDescription TEXT NOT NULL DEFAULT NULL,
+  price NUMERIC(12,2) NOT NULL DEFAULT NULL,
+  nyop BOOLEAN NULL DEFAULT false,
+  min_price NUMERIC(12,2) NULL DEFAULT NULL,
+  availableFrom DATE NOT NULL DEFAULT NULL,
+  availableTo DATE NOT NULL DEFAULT NULL,
+  address1 TEXT NOT NULL DEFAULT NULL,
```

Comment:
I noticed there isn't any addition address fields. Maybe just 'address' instead of 'address1'?

Reply:
address is the reserved word (i think).